js/main.js

```diff
@@ -502,7 +502,7 @@ var MM = (function () {
 			modules = [];
 			for (var m in moduleObjects) {
```

Comment:
WOuldnt this even be easier to read with some es6 syntax like for-each?

Reply:
but why, foreach requires a function, we got all that already..

Reply:
I'm not good in js, so feel free to change ...

Reply:
Would be this line instead of the whole for block:

			moduleObjects.forEach((module) => modules.push(module));